ClientApp/package.json

```diff
@@ -11,8 +11,11 @@
     "@material-ui/core": "^4.12.3",
     "@material-ui/lab": "^4.0.0-alpha.60",
     "@microsoft/signalr": "^5.0.11",
+    "@react-three/drei": "^7.25.1",
+    "@react-three/fiber": "^7.0.21",
     "axios": "^0.22.0",
     "bootstrap": "^4.6.1",
+    "drei": "^2.2.21",
```

Comment:
Are `react-three/drei` and `drei` dependencies linked together or is `drei` not used?

Reply:
Drei is actually not used, removed it.

Reply:
There was also `rimraf` and `three` added which might also be redundant?

Reply:
These are actually needed dependencies for other modules, as far as I know